src/dropdown/dropdown.ts

```diff
@@ -93,12 +126,36 @@ export class NgbDropdown {
     }
   }
 
+  ngAfterContentInit() {
+    this._dropdownMenu = this._elementRef.nativeElement.querySelector('.dropdown-menu');
```

Comment:
I'm not sure how I feel about querying for  `.dropdown-menu` like this... Maybe we should create a directive for this? The benefit would be that we could automatically add `dropdown-menu` class as well as all the required `aria` attributes.